packages/ai/src/ai/modules/task/task_logger.py

```
@@ -0,0 +1,45 @@
+        payload = {
+            'timestamp': time.strftime(
+                '%Y-%m-%dT%H:%M:%SZ', time.gmtime(record.created)
+            ),
+            'level':   record.levelname,
+            'logger':  record.name,
+            'message': record.message,
+        }
+
+        for key, value in record.__dict__.items():
+            if key not in self._RESERVED:
+                payload[key] = value
```

Ruff (0.15.12)

[warning] Use a dictionary comprehension instead of a for-loop

(PERF403)